test/data/helm-chart/template.yaml

```diff
@@ -553,6 +401,275 @@ spec:
               echo "Configuration successful"
         
                 
+        - name: configure-developer-hub
+          image: "quay.io/codeready-toolchain/oc-client-base:latest"
+          command:
+            - /bin/bash
+            - -c
+            - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+            
+        
+        
+              # Installing Helm...
+              curl --fail --silent --show-error --location \
+                https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 \
+                  | bash
+        
+              YQ_VERSION="v4.40.5"
+              curl --fail --location --output "/usr/bin/yq" --silent --show-error "https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64"
+              chmod +x "/usr/bin/yq"
+        
+              CHART="rhtap"
+              NAMESPACE="rhtap"
```

Comment:
Let's use the namespace defined by Helm, i.e.:

https://github.com/redhat-appstudio/rhtap-installer/blob/628a45c1d8ca383bdac34c33975552d5d1ba9134/templates/trusted-profile-analyzer/includes/_configure.tpl#L33-L34

Reply:
This file is just a rendering of the helm chart.  The source has the Helm defined value.